projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -46,7 +46,9 @@ public function reregister_menu_items() {
 			$this->add_new_site_link();
 		}
 
-		$this->add_woocommerce_installation_menu();
+		if ( $this->current_product_is_business_or_higher() ) {
```

Comment:
How will we change this to `premium_or_higher()` later?

Reply:
I renamed that function to be plan agnostic, but we just need to check that the product_id we have is `in_array` of a list of possible product_ids.

`\WPCOM_Store::get_wpcom_business_and_higher_plans();` just returns an array of 6 product_ids. It's nothing fancy. To include Premium plans we just need to include those product_ids. In fact, `\WPCOM_Store::get_wpcom_premium_and_higher_plans();` already exists.